registry/celo/calldata-Election-v1.1.4.0.json

```diff
@@ -0,0 +1,2723 @@
+              },
+              "path": "#.greater",
+              "value": null
+            }
+          ],
+          "required": [
+            "#.group",
+            "#.value"
+          ],
+          "excluded": null
+        },
+        "activate(address)": {
+          "$id": "activate",
+          "intent": "Activate Votes",
+          "screens": null,
+          "fields": [
+            {
+              "$id": null,
+              "label": "Group",
+              "format": "addressName",
+              "params": {
+                "types": [
+                  "wallet",
+                  "eoa",
+                  "contract"
+                ],
+                "sources": null
+              },
+              "path": "#.group",
+              "value": null
+            }
+          ],
+          "required": [
+            "#.group"
+          ],
+          "excluded": null
+        },
+        "activateForAccount(address,address)": {
+          "$id": "activateForAccount",
+          "intent": "Activate Votes For",
+          "screens": null,
+          "fields": [
+            {
+              "$id": null,
+              "label": "Group",
+              "format": "addressName",
+              "params": {
+                "types": [
+                  "wallet",
+                  "eoa",
+                  "contract"
+                ],
+                "sources": [
+                  "ens",
+                  "local"
+                ]
+              },
+              "path": "#.group",
+              "value": null
+            },
+            {
+              "$id": null,
+              "label": "Account",
+              "format": "addressName",
+              "params": {
+                "types": [
+                  "eoa",
+                  "wallet"
+                ],
+                "sources": null
+              },
+              "path": "#.account",
+              "value": null
+            }
+          ],
+          "required": [
+            "#.group",
+            "#.account"
+          ],
+          "excluded": null
+        },
+        "revokePending(address,uint256,address,address,uint256)": {
+          "$id": "revokePending",
+          "intent": "Revoke Pending Votes",
+          "screens": null,
+          "fields": [
+            {
+              "$id": null,
+              "label": "Group",
+              "format": "addressName",
+              "params": {
+                "types": [
+                  "wallet",
+                  "eoa",
+                  "contract",
+                  "token",
+                  "collection"
+                ],
+                "sources": null
+              },
+              "path": "#.group",
+              "value": null
+            },
+            {
+              "$id": null,
+              "label": "Value",
+              "format": "unit",
+              "params": {
+                "base": "",
+                "decimals": "18",
+                "prefix": "CELO"
```

Comment:
`prefix` is a boolean:

> A boolean indicating whether an SI prefix should be appended, defaults to False

I think you are looking for `base`: https://github.com/LedgerHQ/clear-signing-erc7730-registry/blob/master/specs/erc-7730.md?plain=1#L1012